backend/typescript/utilities/logger.ts

```diff
@@ -13,14 +13,32 @@ const logger: winston.Logger = winston.createLogger({
 });
 
 if (process.env.NODE_ENV !== "production") {
-  logger.add(
-    new winston.transports.Console({
-      format: winston.format.combine(
-        winston.format.timestamp(),
-        winston.format.json(),
-      ),
-    }),
+  _logger.add(
+    new winston.transports.Console(),
```

Comment:
The removed format options were redundant right? In the future it'd be nice to make a small PR comment about that 👍

Reply:
Yes the format option was redundant, comment added to PR description.